seedlet/inc/headstart/en.json

```
@@ -74,7 +60,7 @@
             "hs_custom_meta": "_hs_front_page",
             "hs_sharing": 1,
             "hs_like_status": 1,
-            "attachment_url": "https:\/\/headstartdata.files.wordpress.com\/2016\/06\/skyline-buildings-new-york-skyscrapers.jpg",
+            "attachment_url": "https://seedletdemo.files.wordpress.com/2020/06/ec867-plants_2x2.png",
```

```suggestion
            "attachment_url": "https://headstartdata.files.wordpress.com/2020/07/envelope.jpg",
```

I've uploaded a version of this to the headstartdata site. 👍

---

Actually, I'm not sure it even makes a difference where the images come from. 🤷‍♂️

I've updated the theme demo, and will update this PR in just a moment.